.github/workflows/release-prod.yml

```diff
@@ -0,0 +1,39 @@
+name: Release (prod)
+
+on:
+  push:
+    branches: [main]
+    paths: ['modal/**']
+
+jobs:
+  release:
+    name: Release (prod)
+    runs-on: ubuntu-latest
+    env:
+      MODAL_TOKEN_ID: ${{ secrets.MODAL_TOKEN_ID }}
+      MODAL_TOKEN_SECRET: ${{ secrets.MODAL_TOKEN_SECRET }}
```

Comment:
will need to configure some new secrets in the repo settings, I don't have the perms rn

Reply:
Done! cc @alexanderatallah to config @sambarnes 's perm